src/components/ApplicationDashboard/ApplicationDashboard.tsx

```diff
@@ -85,6 +72,15 @@ export const ApplicationDashboard: FC<Props> = props => {
                         if (!application) {
                             return null;
                         }
+
+                        const result: DateandtimeReturnType = {
```

Comment:
You don't need to spread or type this. You can either just use the result:

`const result = daysLeft(application.closeDate)`

 which is identical to the result you have, or deconstruct it:

`const { timeToShow, prefixToShow } = daysLeft(application.closeDate);`